.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -4,43 +4,45 @@ about: Report a bug
 labels: bug, status/needs-triage
 ---
 
-<!-- NOTE:📕 If issue regarding developer guide, please create issue [here](https://github.com/awsdocs/aws-cdk-guide/issues) -->
-
-## :bug: Bug Report
-
-### What is the problem?
 <!--
-Describe in as much detail as possible the problem that you have identified.
-Please identify clearly what construct the problem relates to, or indicate
-what part of the core cdk codebase is causing problems.
+description of the bug:
 -->
 
+
+
+
 ### Reproduction Steps
-<!--
-Whenever possible, add reference to the minimal reproduction of the issue, as
-this will greatly help with identifying the root cause of the problem, and
-ensuring a candidate fix is working.
 
-If your minimal reprodiction is large, you may provide references to a [gist][1]
-or any other publicly viewable location.
+<!--
+minimal amount of code that causes the bug (if possible) or a reference:
 -->
 
-### Verbose Log
-<!--
-If the output is very large, you may store it in a [gist][1]. In any case,
```

Comment:
You dont want to reference Gist here?

Reply:
I haven't seen this as an issue. If we see this becoming an issue we can add a note.